internal/services/datadog/azurerm_datadog_monitor_sso_configuration.go

```diff
@@ -143,10 +170,14 @@ func resourceDatadogSingleSignOnConfigurationsRead(d *pluginsdk.ResourceData, me
 	if model := resp.Model; model != nil {
 		if props := model.Properties; props != nil {
 			// per the create func
-			singleSignOnEnabled := props.SingleSignOnState != nil && *props.SingleSignOnState == singlesignon.SingleSignOnStatesEnable
-			d.Set("single_sign_on_enabled", singleSignOnEnabled)
+			d.Set("single_sign_on", props.SingleSignOnState)
```

Comment:
this will need casting to a string I think? (Set() can't handle the deref and decoding the enum type to string)